game.sy

```diff
@@ -116,6 +117,13 @@ player_new :: fn pos: (float, float), team: int -> Player {
                 p.draw'
                 p.pos -= puck_offsets[segment] * self.rad
             }
+
+            // Arrow
```

Comment:
Can we maybe not draw the arrow for players that we're not currently controlling? Or should we see everyones arrow? :blush:

Maybe make it bigger?